Actions/AL-Go-Helper.ps1

```diff
@@ -966,13 +974,18 @@ function CreateDevEnv {
                 "artifact" = $repo.artifact.replace('{INSIDERSASTOKEN}',$insiderSasToken)
                 "auth" = $auth
                 "credential" = $credential
-                "updateLaunchJson" = "Local Sandbox"
             }
             if ($containerName) {
                 $runAlPipelineParams += @{
+                    "updateLaunchJson" = "Local Sandbox ($containerName)"
                     "containerName" = $containerName
                 }
             }
+            else {
+                $runAlPipelineParams += @{
+                    "updateLaunchJson" = "Local Sandbox"
+                }
+            }

```

Comment:
minor: code can be simplified

Reply:
I don't think so - it is important that the ContainerName is NOT specified in the params if the value is blank - and if I want the ( ) around the containername in local sandbox, I think I need the if.
If this is not what you had in mind, let me know.